src/sas/qtgui/Utilities/GuiUtils.py

```diff
@@ -1239,7 +1239,7 @@ def jdefault(o):
             return add_type(content, type(o))
 
         # ndarray
-        if isinstance(o, np.ndarray):
+        if isinstance(o, (np.ndarray, np.float)):
```

Comment:
`np.float` is just the python `float` type, which does not have a "tolist" attribute:
```
>>> __builtin__.float == np.float
True
>>> np.float(10.).tolist()
AttributeError: 'float' object has no attribute 'tolist'
```
I'm not sure it is meaningful to turn a scalar into list, but `np.float32/np.float64` both define `tolist` to return a scalar builtin floating point value.

So maybe you mean `if isinstance(o, (np.ndarray, np.float32, np.float64):`?

I haven't looked at the context to see what happens if the type condition returns false.